examples/app-service-asp-net/app.tf

```diff
@@ -0,0 +1,65 @@
+# Configure the Microsoft Azure Provider
+provider "azurerm" {
+  subscription_id = "${var.subscription_id}"
+  client_id       = "${var.client_id}"
+  client_secret   = "${var.client_secret}"
+  tenant_id       = "${var.tenant_id}"
+}
```

Comment:
we tend to leave this commented out in case people opt to use Azure CLI auth (I'll push a commit to amend this)